Riku/src/Riku.cpp

```diff
@@ -289,41 +286,32 @@ void drawScene(Shader& lightingShader, Shader& lightCubeShader, float currentFra
 	lightingShader.setVec4("color_mod", 1.0f,1.0f,1.0f, 1.0f);
 	// render the loaded models
 	//draw tiles
-	for(int i=0;i<front::tiles.size();i++)
+	auto& map = front::state.getMap();
+	for(int i=0;i<map.size();i++)
 	{
-		for(int j=0;j<front::tiles[i].size();j++)
+		for(int j=0;j<map[i].size();j++)
 		{
 			front::Object object;
-			if(front::tiles[i][j].type<0 || front::tiles[i][j].type>tileTypeModels.size())
-				object = front::Object(tileTypeModels[0],glm::vec3((float)i,0.0f,(float)j));
+			if(map[i][j].area.getName()=="wet")
+				object = front::Object(front::groundModels["wet"],glm::vec3(i,(float)map[i][j].height*0.5f,j));
 			else
-				object = front::Object(tileTypeModels[front::tiles[i][j].type],glm::vec3(i,front::tiles[i][j].level*0.5f,j));
+				object = front::Object(front::groundModels[map[i][j].ground.getName()],glm::vec3(i,(float)map[i][j].height*0.5f,j));
```

Comment:
Czy to co jest w elsie nie zrobiłoby tego samego co jest w if'ie? Tzn jeśli usunelibyśmy if i zostawilibyśmy tylko to co w elsie to nie działałoby tak samo?